docs/coordinates/matchsep.rst

```diff
@@ -192,13 +193,23 @@ straightforward to find the closest coordinates in a catalog to a desired set
 of other coordinates. For example, assuming ``ra1``/``dec1`` and
 ``ra2``/``dec2`` are numpy arrays loaded from some file::
 
+.. testsetup ::
+
+    >>> ra1= [5.3517]
+    >>> dec1= [-5.2328]
+    >>> distance1 = 1344
+    >>> ra2= [6.459]
+    >>> dec2= [-16.4258]
+    >>> distance2 =8.611
+
     >>> from astropy.coordinates import SkyCoord
     >>> from astropy import units as u
-    >>> c = SkyCoord(ra=ra1*u.degree, dec=dec1*u.degree)  # doctest: +SKIP
-    >>> catalog = SkyCoord(ra=ra2*u.degree, dec=dec2*u.degree)  # doctest: +SKIP
-    >>> idx, d2d, d3d = c.match_to_catalog_sky(catalog)  # doctest: +SKIP
+    >>> import scipy
```

Comment:
remove this import.scipy is neither used nor a mandatory dependency, so this could fail tests.